src/Parsers/XMLParser.php

```diff
@@ -50,7 +50,7 @@ private function searchElement(callable $function)
 		}
 	}
 
-	private function extractElement(String $elementName, $couldBeAnElementsList = false, int $parentDepth)
+	private function extractElement(String $elementName, $couldBeAnElementsList = false, int $parentDepth, ?string $foundInEl = null)
```

Comment:
Let's remove the question symbol as we're making the parameter nullable twice here